radio/src/lv_conf.h

```diff
@@ -559,7 +559,7 @@
 
 #define LV_USE_CANVAS     1
 
-#define LV_USE_CHECKBOX   1
+#define LV_USE_CHECKBOX   0
```

Comment:
Is this not used somewhere?
I think the model checklist PR does use these.

Reply:
Thanks. I will restore the CheckBox class and re-enable the LVGL CHECKBOX feature.

Reply:
The CheckBox class is actually using an LVGL SWITCH, not an LVGL CHECBOX - just to make things confusing :)

The checklist PR looks to be using LVGL checkboxes so we will have both types in future.

I will probably rename the CheckBox class.

Reply:
Sounds good to me. The wide switched used for checkboxes did seem a bit bulky to me for the checklist PR, so I used the checkboxes.